FinniversKit/Sources/Components/HTMLLabel/HTMLLabel.swift

```diff
@@ -1,26 +1,31 @@
 public class HTMLLabel: Label {
     private var htmlText: String?
-    private var styleMap: HTMLStyler.StyleMap
-    private var additionalStyleMap: HTMLStyler.StyleMap?
+    private let spanMapper: HTMLStringUIKitStyleTranslator.SpanMapper
```

Comment:
In the future this class should probably be modified to accept custom mapping for any element. We have no idea what kinds of HTML the various backends want to send.

Reply:
There are various weights of bold, and in CSS you can style elements however you want regardless of their original intent. We need a default for the common semantic tags though.

Reply:
That's fine, but until we don't get that sophisticated HTML syntax from backend, I think it's simpler to only open up `span` at the moment.